stock_lot_name_adjust/views/stock_lot_views.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Can we remove this file?  I now do not think this improves the search usability.